pkg/summary/options.go

```diff
@@ -6,21 +6,20 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
-// Functional options
-type Option func(*Options)
+type option func(*options)
```

Comment:
This should not have been unexported. It's used in the `cmd` package. Once it's exported again you'll need to update the references in the the `summary` package as well.

Reply:
Whoops! Nice catch, sorry for that!